### PR TITLE
{phys}[intel/2019b] BLOCK v1.5.3 w/ Python 2.7.16

### PR DESCRIPTION
**Pull request by nicholascheng94**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by nicholascheng94** [comment]
Test report by @nicholascheng94
**SUCCESS**
Build succeeded for 1 out of 1 (2 easyconfigs in total)
sms - Linux CentOS Linux 7.6.1810, x86_64, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/fc4977d7ba1a4b545656917138e8c564 for a full test report.

File: easybuild/easyconfigs/b/BLOCK/BLOCK-1.5.3-intel-2019b-Python-2.7.16.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://sanshar.github.io/Block/'
+description = """BLOCK implements the density matrix renormalization group (DMRG) algorithm for quantum chemistry."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'cstd': 'c++11'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.sunqm.net/pyscf/files/src/']
```
Comment (lexming) [inline easybuild/easyconfigs/b/BLOCK/BLOCK-1.5.3-intel-2019b-Python-2.7.16.eb:14]: It is necessary to add the checksums of all sources and files. You can do so with the command `eb --inject-checksums`

**Review by lexming — CHANGES_REQUESTED** [review]

**Review by lexming — COMMENTED** [review]
@nicholascheng94 I've been also working on Block v1.5.3 as well, but in a more recent toolchain. See #16711.

I made a PR to your fork with my suggested changes. Please check https://github.com/nicholascheng94/easybuild-easyconfigs/pull/1 and, if you agree with the changes, merge it to automatically update this PR.

This is the list changes compared to your easyconfig:
* use sources from new repo in https://github.com/pyscf/Block
* add missing checksums (with `eb --inject-checksums`)
* use Python only as `builddependency` (it is not needed to execute `block.spin_adapted`)
* use version of Boost provided in this toolchain (see https://github.com/easybuilders/easybuild-easyconfigs/pull/16642#discussion_r1025895601)
* build both the MPI and serial versions of Boost
* add sanity check commands

**Comment by nicholascheng94** [comment]
Thanks @lexming ! So sorry for the delay... was not working on using this software anymore so had not the time to get back on this.

**Comment by boegel** [comment]
Closing this since `intel/2019b` is no longer supported, see https://docs.easybuild.io/policies/toolchains

We have an easyconfig for Block with a more recent toolchain, see #19493

**Pull request closed without merging**